src/main/java/carracing/domain/Cars.java

```diff
@@ -2,53 +2,43 @@
 
 import carracing.util.RandomUtil;
 
-
 import java.util.ArrayList;
 import java.util.Arrays;
-import java.util.Comparator;
 import java.util.List;
 import java.util.stream.Collectors;
 
 public class Cars {
 
     private List<Car> cars = new ArrayList<>();
 
-    public Cars(String carNames) {
-        saveCarNames(carNames);
+    public Cars(String[] carNames) {
+        Arrays.stream(carNames).forEach(name -> this.cars.add(new Car(name)));
     }
 
-    public List<Car> getCars() {
-        return this.cars;
+    public Cars(List<Car> carList) {
+        this.cars = carList;
```

Comment:
```suggestion
    public Cars(List<Car> cars) {
        this.cars = cars;
```
컬렌션의 경우 복수형 명사로 사용해주는게 좋습니다! 그래야 조금 확장성 있는 이름이 되기 때문입니다.